### DISCUSSION TITLE
About data integration

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#827] [Questions / Documentation]
Thanks for your ArchR！

Why is it that using ArchR to integrate scATAC data does not integrate well and each sample is separated from each other?

The seurat integration of RNA data allows for good integration between different samples.

Is this the cause of the data type?

ATAC(using ArchR)
![image](https://user-images.githubusercontent.com/54537420/122354723-d9ff8880-cf83-11eb-9956-bb803d1209aa.png)

RNA(using Seurat)
![image](https://user-images.githubusercontent.com/54537420/122354757-e2f05a00-cf83-11eb-9efc-e6757428b8b7.png)

**Comment by rcorces** [comment]
Thanks for using ArchR. I'm afraid that your question doesnt really have an answer. snATAC-seq data from different samples can be very effectively integrated, which we show in our publication combining datasets generated across multiple different platforms (Fluidigm, biorad, 10x Genomics) etc. If you are having trouble, perhaps your data is highly confounded by batch effects or you need to tune the parameters of your dimensionality reduction / try using Harmony.

**Reply by tzhu-bio** [reply]
Thanks for your reply!
When I use addHarmony, I got this message:
"Harmony converged after 3 iterations"
And I found no obvious change in the results.

**Reply by rcorces** [reply]
I'm sorry that your analysis is not meeting your expectations but there is no further help I can provide. I can assure you that this is not a problem with ArchR.
